offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/products/MaintainProductsView.groovy

```diff
@@ -0,0 +1,24 @@
+package life.qbic.portal.offermanager.components.products
+
+import com.vaadin.ui.FormLayout
+
+/**
+ *
+ * <h1>This class generates a Form Layout in which the user can maintain the service products</h1>
```

Comment:
```suggestion
 * <h1>This class generates a Vertical Layout in which the user can maintain the service products</h1>
```

Reply:
@jenniferboedker

Reply:
I think this is the responsibility of #392 and stay as is. It will get resolved when the other PR is merged